pyflakes/checker.py

```diff
@@ -2383,3 +2383,10 @@ def COMPARE(self, node):
             left = right
 
         self.handleChildren(node)
+
+    MATCH = MATCH_CASE = handleChildren
+    MATCHCLASS = MATCHMAPPING = MATCHSEQUENCE = MATCHVALUE = handleChildren
+
+    def MATCHAS(self, node):
+        self.handleNodeStore(node)
+        self.handleChildren(node)
```

Comment:
ideally I wouldn't need this -- opened https://bugs.python.org/issue43994

Reply:
I noticed that as well for pylint / astroid. Would have been nice to have, but I guess we now have to wait till 3.11 for it 🤷🏻‍♂️